### PR TITLE
7466 - adding logic to support firmName updates

### PR DESCRIPTION
**Pull request by codyseibert**
Displaying the Firm Name input box for privatePractitioners on the edit contact information page.  This required adding in some logic to the proxy & interaction to handle updates to only the firmName by itself so that it didn't go and generate changeOfAddress pdfs.

Screen Shot 2021-04-15 at 5 16 28 PM

### CODE REVIEW

**Comment by codyseibert** [comment]
Any clue why this only ran 3 checks?  It didn't run integration tests or anything else.

File: shared/src/business/useCases/users/updateUserContactInformationInteractor.test.js [hunk 1]
```diff
@@ -304,4 +305,56 @@ describe('updateUserContactInformationInteractor', () => {
...
+    await updateUserContactInformationInteractor(applicationContext, {
+      contactInfo,
+      firmName: mockUser.firmName,
+      userId: 'f7d90c05-f6cd-442c-a168-202db587f16f',
+    });
+
+    expect(
+      applicationContext.getPersistenceGateway().updateUser,
+    ).not.toBeCalled();
+  });
```
Comment (mmarcotte) [inline shared/src/business/useCases/users/updateUserContactInformationInteractor.test.js]: Do my eyes deceive me, or is this a duplicate test?

**Review by mmarcotte — CHANGES_REQUESTED** [review]
Looks like one dupe test in there.

File: shared/src/business/useCases/users/updateUserContactInformationInteractor.test.js (hunk 1 above)
Comment (codyseibert) [reply shared/src/business/useCases/users/updateUserContactInformationInteractor.test.js]: I think so, fixing now.

**Review by mmarcotte — APPROVED** [review]

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**